@aadattoli additional changes to diagnostics for merge (#181)
greenelab/covid19-review: content/10.diagnostics.md

Following the first release of the genetic sequence of the virus by Chinese officials on January 10 2020, the first test was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045]. 
Testing who contracted the infection is crucial to slow down the global pandemic. The available assays use the following approach to identify the active virus in patient samples.

Molecular tests

> Molecular tests are often carried out from the nasopharyngeal cavity samples, including throat wash and saliva [doi.org/10.1016/S1473-3099(20)30196-1].

Reviewer: ```suggestion
Molecular tests are often carried out from the nasopharyngeal cavity samples, including throat wash and saliva [@doi:10.1016/S1473-3099(20)30196-1].
```

1) RT-PCR 

The Drosten Lab, from Germany, is the first lab that established and validated the first diagnostic test to detect SARS-Cov-2. This test is based on the real-time PCR (RT-PCR) [doi.org/10.2807/1560-7917.ES.2020.25.3.2000045]. 
Particularly, this test detects several regions of the viral genome: orf1b of the RNA dependent RNA polymerase (RdRP), Envelope protein gene (E), Nucleocapsid protein gene (N). 
The lab, which collaborated with several other labs in Europe and in China, confirmed the specificity of this test, with the respect of other corona viruses, against specimens from 297 patients with a broad range of respiratory agents. 
Specifically this tests utilises two probes against RdRP of which one is specific for COVID-19 [doi.org/10.2807/1560-7917.ES.2020.25.3.2000045]. Importantly, this assay did not give any false positive outcome.

2) qRT-tPCR

> Around the same time the Hong Kong university developed a Quantitative Real Time Reverse Transcription PCR (qRT-tPCR) against two region of the virus, orf1b and N region [doi.org/10.1093/clinchem/hvaa029]. 

Reviewer: ```suggestion
Around the same time the University of Hong Kong developed quantitative real-time reverse transcription PCR (qRT-PCR) assays against two region of the virus, orf1b and N region [@doi:10.1093/clinchem/hvaa029]. 
```

Specifically, this assay was tested on samples coming from two SARS-Cov-2 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses. 
The assay uses the N gene to screen patients, while the orf1b is used to confirm the infection [doi.org/10.1093/clinchem/hvaa029].
In this case the test was designed to detect sabercoviruses of which COVID-19 belongs. 
Considering that no other sabercoviruses are known to infect human at the moment, if the test comes out positive it means the patient is infected with the virus. 
However, this test does not allow to discriminate the genetics among different viruses. 

3) CRISPR test

> Recently, two American companies, Mammoth Biosciences and Sherlock Biosciences, proposed to use CRISPR technology [10.1016/j.cell.2014.05.010]to implement the speed of Coronavirus diagnostic test [doi.org/10.1101/2020.03.06.20032334; doi: 10.1038/s41596-019-0210-2]. 

Reviewer: ```suggestion
Recently, two American companies, Mammoth Biosciences and Sherlock Biosciences, proposed to use CRISPR technology [@doi:10.1016/j.cell.2014.05.010] to implement the speed of Coronavirus diagnostic test [@doi:10.1101/2020.03.06.20032334; @doi: 10.1038/s41596-019-0210-2]. 
```

Their methodology includes purification of RNA extracted from patient specimen; loop-mediated iso-thermal amplification (fast amplification) of the RNA; addition of the CRISPR reagents: CAS12, which cut the DNA or RNA, and the RNA guidance. 
The RNA guidance helps CAS12 to direct the cut of a sequence corresponding to a piece of the coronavirus genome (specifically, N, E and RdRP genes in the test designed by Mammoth). 

> With this test, If CAS12 and its guide find a match in the RNA, then the cut occurs and the patient is positive to the novel Coronavirus. Specifically, when CAS12 is active it also cuts a designed strand containing molecules that change color once they are free [doi.org/10.1101/2020.03.06.20032334]. 

Reviewer: ```suggestion
With this test, if CAS12 and its guide find a match in the RNA, then the cut occurs and the patient is positive to the novel Coronavirus. Specifically, when CAS12 is active it also cuts a designed strand containing molecules that change color once they are free [@doi:10.1101/2020.03.06.20032334]. 
```

> Mammoth CRISPR test is designed to give a practical solution in area that are at greater risk of infection (i.e. airports, local community hospitals). This test shows a sensitivity range of 70-300 copies of the target RNA/ul. 

Reviewer: ```suggestion
Mammoth CRISPR test is designed to give a practical solution in areas that are at greater risk of infection (i.e. airports, local community hospitals). This test shows a sensitivity range of 70-300 copies of the target RNA/ul. 
```

Specifically, the Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay designed to qualitatively detect IgM and IgG antibodies against SARS-Cov-2 in the plasma of patients (blood sample) suspected to have developed the infection [https://www.fda.gov/media/136625/download]. 
Such test allows to understand the progress of the viral disease. Specifically, IgM are the first antibodies produced by the body and indicate that the infection is in action. 
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [doi.org/10.1002/jmv.25820]. 
The test cassette contains a pad of SARS-Cov-2 antigens, a ntrocellulose strip with an IgG line, IgM line and a control line (goat IgG) [https://www.fda.gov/media/136625/download].
When the patient's specimen contains antibody against COVID-19 antigens they will bind to them and capture either by the IgM and or IgG line, with the result of a change of color [https://www.fda.gov/media/136625/download]. 
With this particular assay results can be read within 15-20 minutes [https://www.fda.gov/media/136625/download].

> Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus [doi.org/10.1101/2020.03.17.20037713]. 

Reviewer: ```suggestion
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713]. 
```

### Limitations to Implementation of Large-Scale Testing

Limitation of molecular tests

-PCR tests

The PCR test can be highly sensitive. However, in high-throughput settings it can show several problems:

> 1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [doi: 10.1038/d41587-020-00002-2]. 

Reviewer: ```suggestion
1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [@doi:10.1038/d41587-020-00002-2]. 
```

> 1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [doi: 10.1038/d41587-020-00002-2]. 

Reviewer: ```suggestion
1)	 False-negative responses, which can be a significant problem in a large scale test. In this case a correct execution is crucial [@doi:10.1038/d41587-020-00002-2]. 
```

> 2)	Uncertainty of the SARS-Cov-2 viral shedding kinetics, which may affect the result of test if not taken at the right time [doi: 10.1038/d41587-020-00002-2].

Reviewer: ```suggestion
2)	Uncertainty of the SARS-CoV-2 viral shedding kinetics, which may affect the result of the test if not taken at the right time [@doi:10.1038/d41587-020-00002-2].
```

> 3)	Type of Specimen, as it is not clear which clinical sample is best to detect the virus [doi: 10.1038/d41587-020-00002-2].

Reviewer: ```suggestion
3)	Type of Specimen, as it is not clear which clinical sample is best to detect the virus [@doi:10.1038/d41587-020-00002-2].
```

> 4)	Expensive machinaries, which might be present in major hospital and or diagnostic centers, but not in smaller facilities [doi:10.1126/science.abb8400]. 

Reviewer: ```suggestion
4)	Expensive machinary, which might be present in major hospital and diagnostic centers, but not in smaller facilities [@doi:10.1126/science.abb8400]. 
```

> 5)	Timing of the test, which might take up to 4 days, due to shipping of collected samples, to give results [doi:10.1126/science.abb8400].

Reviewer: ```suggestion
5)	Timing of the test, which might take up to 4 days, due to shipping of collected samples, to give results [@doi:10.1126/science.abb8400].
```

-CRISPR test

Fake-positives due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.

Finally, molecular tests can detect only people that are currently infected, while they cannot detect someone who had developed the infection but has had since recovered.

Limitation of seriological tests

Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [https://www.fda.gov/media/136625/download]. 
Therefore, these tests should be used in combination with RNA detection tests [https://www.fda.gov/media/136625/download]. 

### Strategies and Considerations for Determining Whom to Test

> Currently, Coronavirus tests are limited for people that are in danger of serious illness [https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html]. Specififcally, the individuals at risk include:

Reviewer: ```suggestion
Currently, Coronavirus tests are limited to people that are in danger of serious illness [https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html]. Specifically, the individuals at risk include:
```

-people with severe symptoms

> -people showing mild symptomps that have been in contact with a person who has developed the infection

Reviewer: ```suggestion
-people showing mild symptoms that have been in contact with a person who has developed the infection
```

-people with underlying health conditions
-pregnant women

> However, this methodology of testing administration does not detect a high proportion of infections. This includes individuals that are asymptomatic (potentially able to infect others) and individuals that are able to recover at home. Furthermore, individuals for whom the origin of the infection is uncertain are not tested. For instance, a recent study from the Imperial College estimates that in Italy the true number of infections is around 5.9 million against the 70000 detected as of March 28th [doi.org/10.25561/77731].   

Reviewer: ```suggestion
However, this methodology of testing administration does not detect a high proportion of infections. This includes individuals that are asymptomatic (potentially able to infect others) and individuals that are able to recover at home. Furthermore, individuals for whom the origin of the infection is uncertain are not tested. For instance, a recent study from the Imperial College estimates that in Italy the true number of infections is around 5.9 million against the 70000 detected as of March 28th [@doi:10.25561/77731].   
```
